server/tests/usertest.js

```diff
@@ -0,0 +1,222 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../server';
+import User from '../models/userModel';
+import '../../config';
+
+
+const users = User.users;
+const { expect } = chai;
+chai.should();
+chai.use(chaiHttp);
+
+describe('/Auth', () => {
+    describe('/POST signup', () => {
+        it('should successfully sign up a user', (done) => {
+            chai.request(app)
+                .post('/api/v1/auth/signup')
+                .send({
+                    firstname: 'sharon',
```

Comment:
@sharonandisi please create a separate folder in your test folder and name it mockData inside it create a file for example called i.e userMocks.js. inside it, you will create instances of data to be used in your test file. It is not ethical to write the data in the test file. rather import the data from another file. please use this in all scenarios including for tokens.

Reply:
Alright. Noted. Am on it

Reply:
I don't know why one test is failing on travis while they all pass locally . Let me look into it as I make the changes you have sited